server/chat-commands/room-settings.ts

```diff
@@ -915,6 +914,7 @@ export const commands: ChatCommands = {
 		if (room.persist && isGroupchat) return this.errorReply(`This isn't a groupchat.`);
 		if (!room.persist && !isGroupchat) return this.errorReply(`Use /renamegroupchat instead.`);
 		if (isGroupchat) {
+			if (!user.can('mute', null, room) && !this.can('editroom', null, room)) return false;
```

Comment:
```suggestion
			if (!user.can('lock') && !this.can('editroom', null, room)) return false;
```

Probably don't give roomdrivers the ability, if it comes up?